### DISCUSSION TITLE
Way to dynamically build the list in the ComboBox

### DISCUSSION THREAD

**Discussion by mgrivera** [microsoft/fluentui-blazor#2888] [Q&A] [answered]
Hi guys, 
I wonder if there is a way for the combobox to build its list dynamically using the text the user types in its textbox. 
In the AutoComplete there is the `OnOptionsSearch` event callback. It works super fine and you can build the list just from the text that the user is typing. 
So, is there a way to use something like that in the ComboBox? 
Or, there would be something like that for the Combo in the future? 
Many thanks and bye ...

**Comment by dvoituron** [comment, marked as answer]
Hello,
No, that's what the **FluentAutocomplete** component is for.
There are no plans to include a similar function in the List or Combobox components.

**Reply by mgrivera** [reply]
Thanks @dvoituron
